### PR TITLE
Add the introducation to StronglyTypedId

### PR DESCRIPTION
**Pull request by yangzhongke**
Add the introduction to LessCode.EFCore.StronglyTypedId

As proposed on [https://github.com/dotnet/EntityFramework.Docs/issues/4160#issuecomment-1327216363](https://github.com/dotnet/EntityFramework.Docs/issues/4160#issuecomment-1327216363)

### CODE REVIEW

File: entity-framework/core/extensions/index.md [hunk 1]
```diff
@@ -342,6 +342,17 @@ For EF Core: 6, 7.
...
+### LessCode.EFCore.StronglyTypedId
+
+A source generator that can generate strongly-typed-id classes automatically for entities in Entity Framework Core.
+
+With this extension, developers just need to add [HasStronglyTypedId] on the entity type, all the related classes will be generated automatically at compile time.
+
+For EF Core: 7.
+
+[GitHub repository]([https://github.com/R4ND3LL/EntityFrameworkRuler/](https://github.com/yangzhongke/LessCode.EFCore.StronglyTypedId))
+
```
Comment (ErikEJ) [inline entity-framework/core/extensions/index.md:354]: Remove extra line

File: entity-framework/core/extensions/index.md [hunk 2]
```diff
@@ -342,6 +342,17 @@ For EF Core: 6, 7.
 
 [GitHub repository](https://github.com/R4ND3LL/EntityFrameworkRuler/) | [CLI Tool NuGet](https://www.nuget.org/packages/EntityFrameworkRuler/) | [Design NuGet](https://www.nuget.org/packages/EntityFrameworkRuler.Design/)
 
+### LessCode.EFCore.StronglyTypedId
+
+A source generator that can generate strongly-typed-id classes automatically for entities in Entity Framework Core.
+
+With this extension, developers just need to add [HasStronglyTypedId] on the entity type, all the related classes will be generated automatically at compile time.
```
Comment (ErikEJ) [inline entity-framework/core/extensions/index.md:349]: remove this sentence, and reduce to:

A source generator that can generate strongly-typed-id classes automatically for entities. For EF Core: 7.

File: entity-framework/core/extensions/index.md [hunk 3]
```diff
@@ -342,6 +342,17 @@ For EF Core: 6, 7.
...
+
+For EF Core: 7.
+
+[GitHub repository]([https://github.com/R4ND3LL/EntityFrameworkRuler/](https://github.com/yangzhongke/LessCode.EFCore.StronglyTypedId))
```
Comment (ErikEJ) [inline entity-framework/core/extensions/index.md:353]: Fix link

**Review by ErikEJ — CHANGES_REQUESTED** [review]

**Comment by yangzhongke** [comment]
Thank you, I have resolved all the suggestions.
It shows 'Merging is blocked', what can I do to make it merged?

**Review by yangzhongke — COMMENTED** [review]
Done

File: entity-framework/core/extensions/index.md (hunk 1 above)
Comment (roji) [inline entity-framework/core/extensions/index.md:354]: Can you please remove on of these blank lines? The markdown linter is complaining about it.

**Pull request closed without merging**